ui/printer.py

```diff
@@ -117,7 +118,34 @@ def exit_printer():
         i.set_callback("KEY_LEFT", exit_printer)
         i.set_callback("KEY_ENTER", exit_printer)
         i.listen()
+    image_width, image_height = image.size
+    size = o.width, o.height
+    width = o.width/image_width
```

Comment:
Isn't this integer division in Python? Any edgecases?

Reply:
Also, any better way to name `width` and `height`? For now, the code is harder to understand because of it